pkg/sdk/helper.go

```diff
@@ -111,6 +111,7 @@ func (h *Helper) API(serviceModelName string) (*model.SDKAPI, error) {
 		sdkapi := model.NewSDKAPI(api, h.APIGroupSuffix)
 
 		h.InjectCustomShapes(sdkapi)
+		h.InjectSimpleSchemaShapes(sdkapi)
```

Comment:
Shall we make those one function? At least i believe we need to inject custom structs before we start injecting `map_of` and `list_of`